src/vm/diagnosticprotocolhelper.h

```diff
@@ -8,15 +8,40 @@
 #ifdef FEATURE_PERFTRACING
 
 #include "common.h"
+#include <diagnosticsprotocol.h>
+
 
 class IpcStream;
 
+// The Diagnostic command set is 0x01
+enum class DumpCommandId : uint8_t
+{
+    // reserved      = 0x00,
+    GenerateCoreDump = 0x01,
+    // future
+};
+
+struct GenerateCoreDumpCommandPayload
+{
+    // The protocol buffer is defined as:
+    //   string - dumpName (UTF16)
+    //   int - dumpType
+    //   int - diagnostics
```

Comment:
What does the `diagnostics` integer mean?

Reply:
I think you meant to reply here @mikem8361:

> It's actually a bool and if true turns on createdump logging.